specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -122,9 +122,8 @@ function TreeView<SCHEMA extends AnyTree>({
   const toolbarButtonRef = React.useRef<HTMLAnchorElement | null>(null);
   const [isEditingRanks, _, __, handleToggleEditingRanks] = useBooleanState();
 
-  const [screenWidth, setScreenWidth] = React.useState(window.innerWidth);
   const [rawIsSplit = false, setRawIsSplit] = useCachedState('tree', 'isSplit');
-  const canSplit = screenWidth >= 640;
+  const [canSplit, setCanSplit] = React.useState(window.innerWidth >= 640);
```

Comment:
(optional) 

Can you extract `640` into its own variable? 
something like `const minimumSplitWidth = 640` and use that everywhere we use 640. 

Even better if you put it outside of the JSX React function at the top level of the module and use the standard global variable naming convention in Specify:
Something like
`const SPLIT_MIN_SCREEN_WIDTH = 640`

See below for an example:

https://github.com/specify/specify7/blob/b63632a7eb9d247b879023d7e19ca4fba07a567c/specifyweb/frontend/js_src/lib/components/Notifications/hooks.tsx#L11-L13